### PR TITLE
Add vertical table handling

### PR DESCRIPTION
**Pull request by daved**
https://www.pivotaltracker.com/story/show/176045967

 https://www.pivotaltracker.com/story/show/176045967

### CODE REVIEW

File: internal/output/plain.go [hunk 1]
```diff
@@ -28,6 +28,10 @@ type PlainOpts string
 const (
 	// SeparateLineOpt requests table output to be printed on a separate line (without columns)
 	SeparateLineOpt PlainOpts = "separateLine"
+	// VerticalTable requests a table be output vertically
+	VerticalTable PlainOpts = "tableVert"
```
Comment (Naatan) [inline internal/output/plain.go:32]:
```suggestion
	// VerticalTable requests a table be output vertically
	VerticalTable PlainOpts = "verticalTable"
```

File: internal/output/plain.go [hunk 2]
```diff
@@ -28,6 +28,10 @@ type PlainOpts string
...
+	// HorizontalTable requests a table be output horizontally (default for slices of structs)
+	HorizontalTable PlainOpts = "tableHoriz"
```
Comment (Naatan) [inline internal/output/plain.go:34]: This is the default, don't need an option for this.
```suggestion
```

File: internal/table/table.go [hunk 3]
```diff
@@ -188,3 +219,25 @@ func pad(v string) string {
 	padded := strings.Repeat(" ", padding)
 	return padded + v + padded
 }
+
+func verticalRows(hdrs []string, rows []row) []row {
```
Comment (Naatan) [inline internal/table/table.go:223]: Please undo all this stuff to the table package. As the ACs state the only change here is disregarding the headers.

**Review by Naatan — CHANGES_REQUESTED** [review]

**Review by Naatan — APPROVED** [review]

**Comment by daved** [comment]
A followup story is filed here: https://www.pivotaltracker.com/story/show/176100516

**Review by Naatan — APPROVED** [review]

**Pull request merged**